Formula/krew.rb

```diff
@@ -39,6 +39,13 @@ def install
     bin.install "build/krew" => "kubectl-krew"
   end
 
+  def caveats
+    <<~EOS
+      By default krew will install binaries in $HOME/.krew/bin. You can add this directory to your PATH with:
+        PATH="$HOME/.krew/bin:$PATH"
+    EOS
+  end
```

Comment:
It's documented upstream: https://krew.sigs.k8s.io/docs/user-guide/setup/install/

Reply:
The [upstream documentation](https://krew.sigs.k8s.io/docs/user-guide/setup/install/) lists the PATH modification as part of its install. I understand that brew will not manipulate PATH so I think brew should at least print a caveat that this part of the install is not automated

The caveat is not so much about modifying the install path, [that's well documented](https://krew.sigs.k8s.io/docs/user-guide/advanced-configuration/). It's about providing a functional installation of krew.

Reply:
I've changed the wording, though I'm still not sure what the issue is. Let me know if you require further changes

Reply:
The issue is that very few people read the caveats, mostly because brew already outputs a lot of information. So we should limit the information to just the information that is specific to Homebrew. If upstream documents something, Homebrew must not document it.